content/en/docs/tasks/observability/distributed-tracing/zipkin/multicluster_test.sh

```diff
@@ -21,14 +21,18 @@ set -o pipefail
 
 # @setup multicluster
 
+source "tests/util/helpers.sh"
```

Comment:
This isn't needed because helpers.sh is one of the auto included files: https://github.com/istio/istio.io/tree/master/tests#include-files